File: Uva/UVa10929.py
while True:
    num = str(input())

    if num == '0':
        break
    even = 0; odd = 0;
    for i in range(len(num)-1, -1, -1):
        if i % 2 == 0:
            even += int(num[i])
            # print(f'{even=}')
        else:
            odd += int(num[i])
            # print(f'{odd=}')
    if (even - odd) % 11 == 0:
        print('{} is a multiple of 11.'.format(num))
        
    else:
        print('{} is not a multiple of 11.'.format(num))

File: UVa_cpp/UVa10929.cpp
#include <iostream>

using namespace std;

int main(){
    string s;
    while(cin >> s){
        if ( s == "0") break;
        int even = 0, odd = 0;
        for(int i=0; i<s.size(); i++){
            if (i % 2 == 0)
                even += s[i] - '0';
            else 
                odd += s[i] - '0';
        }

        if (abs(odd - even) % 11 == 0) cout << s << " is a multiple of 11.\n";
        else cout << s << " is not a multiple of 11.\n";
    }
}